Woof/WoofTests/ViewModel/OwnerMainTabViewModelTests.swift

```diff
@@ -1,18 +1,56 @@
 import XCTest
 
 final class OwnerMainTabViewModelTests: XCTestCase {
-    func testResetCurrentRoleMethodExistsInAPI() {
-        OwnerMainTabViewModel().resetCurrentRole()
+    override func setUp() {
+        super.setUp()
+        viewModel = OwnerMainTabViewModel()
     }
 
-    func testResetCurrentRoleChangesUserRoleToNone() {
-        // Given
-        PreferencesHandler.set(userRole: .owner)
+    func testIsAlertShownPropertyExistsInAPI() {
+        _ = viewModel.isAlertShown
+    }
 
-        // When
-        OwnerMainTabViewModel().resetCurrentRole()
+    func testIsAlertShownPropertyReturnsAssignedValue() {
+        // Given // When
+        viewModel.isAlertShown = true
 
         // Then
-        XCTAssertEqual(PreferencesHandler.getUserRole(), Role.none)
+        XCTAssertTrue(viewModel.isAlertShown)
+
+        // Given // When
```

Comment:
It is worth splitting the test into two separate ones. The current structure is confusing.

Reply:
We check the property to return the assigned value. 
Assign "true" - check.
Assign "false" - check.
Why does it confuse you? I don't quite understand.

Reply:
Agree with @iBYarchitect

Reply:
ok